server-worker/src/main/java/com/aliyun/emr/rss/service/deploy/worker/Writer.java

```diff
@@ -392,4 +398,20 @@ public long getSplitThreshold() {
   public PartitionSplitMode getSplitMode() {
     return splitMode;
   }
+
```

Comment:
This section is due to scala 2.11 restriction, the trait can not be treated as an interface with default methods.